pkg/endpoint/bpf.go

```diff
@@ -558,8 +558,18 @@ func (e *Endpoint) regenerateBPF(owner Owner, epdir, reason string) (uint64, err
 	owner.GetCompilationLock().RLock()
 	defer owner.GetCompilationLock().RUnlock()
 
+	buildStart := time.Now()
+
 	e.Mutex.Lock()
 
+	e.getLogger().Info("Regenerating BPF program")
+	defer func() {
+		e.Mutex.RLock()
+		e.getLogger().WithField(logfields.BuildTime, time.Since(buildStart).String()).
+			Info("Regeneration of BPF program has completed")
```

Comment:
The absolute start time might be nice here. We keep getting logs without timestamps, and sneaking this one in might be handy to correlate when endpoints began building relative to each other (if we're trying to infer if the build queue is backed up, for example).